README.md

```diff
@@ -38,20 +38,20 @@ let establishData = {
 };
 ```
 
-After that, when an Oauth bank is selected in the Trustly Lightbox, a message will be triggered and must be captured by the `onMessage` attribute of the WebView, as in the example below (and in the App.tsx file):
+After that, when an OAuth bank is selected in the Trustly Lightbox, a message will be triggered and must be captured by the `onMessage` attribute of the WebView, as in the example below (and in the `App.tsx` file):
 
 ```js
 <WebView
   ...
-  onMessage={this.handleOauthMessage}
+  onMessage={this.handleOAuthMessage}
   ...
 />
 ```
 
-The function informed in `onMessage` will receive the message with a url that must be opened in an in-app-browser, as in the example below (and in the App.tsx file):
+The function informed in `onMessage` will receive the message with a URL that must be opened in an in-app browser, as in the example below (and in the `App.tsx` file):
```

Comment:
The function provided in `onMessage` will receive a message containing a URL, which must be opened in an in-app browser, as shown in the example below and in the `App.tsx` file: